.gitignore

```diff
@@ -0,0 +1,159 @@
+### macOS
```

Comment:
Using `.gitignore` is always recommended, it helps avoid accidentally adding/commiting files we don't want to be a part of our repo. `.gitignore` is project/repo specific, while `.gitignore_global` is obviously for all git projects.